tests/src/test/java/ca/bc/gov/open/jag/efiling/services/SubmissionService.java

```diff
@@ -0,0 +1,74 @@
+package ca.bc.gov.open.jag.efiling.services;
+
+import ca.bc.gov.open.jag.efiling.error.EfilingTestException;
+import ca.bc.gov.open.jag.efiling.helpers.SubmissionHelper;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.MultiPartSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.UUID;
+
+public class SubmissionService {
+
+    @Value("${EFILING_HOST:http://localhost:8080}")
+    private String eFilingHost;
+
+    private static String CONFIG_PATH = "config";
+
+    private Logger logger = LoggerFactory.getLogger(SubmissionService.class);
+
+    public Response documentUploadResponse(UUID transactionId, String universalId, String accessToken, MultiPartSpecification fileSpec) {
+
+        logger.info("Submitting document upload request to the host {}", eFilingHost);
+
+        Response documentResponse = SubmissionHelper.uploadADocumentRequest(transactionId, accessToken, universalId,

```

Comment:
The `submissionHelper` class does not accomplish much, you can bring the code back in this class. it will be easier to maintain. if you want to pull out some code from a method, you can start by adding a private method inside the same class.

Reply:
Fixed. I understand the redundancy now. Thanks @alexjoybc!